internal/util/logging/logram_test.go

```diff
@@ -0,0 +1,120 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logging
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/zap/zapcore"
+)
+
+func TestLogRAM(t *testing.T) {
+	for name, tc := range map[string]struct {
+		size     int64
+		msgPanic string
+	}{
+		"PanicNegativSize": {
+			size:     -2,
+			msgPanic: "logram size -2",
+		},
+		"PanicZeroSize": {
+			size:     0,
+			msgPanic: "logram size 0",
+		},
+	} {
+		name, tc := name, tc
+		t.Run(name, func(t *testing.T) {
+			assert.PanicsWithValue(t, tc.msgPanic, func() { NewLogRAM(tc.size) })
+		})
+	}
+
+	logram := NewLogRAM(2)
+	for n, tc := range []struct {
+		inLog    zapcore.Entry
+		expected []zapcore.Entry
+	}{
+		{
+			inLog: zapcore.Entry{
+				Level:      1,
+				Time:       time.Date(2022, 12, 31, 11, 59, 1, 0, time.UTC),
+				LoggerName: "logger_1",
+				Message:    "message 1",
+			},
+			expected: []zapcore.Entry{
+				{
+					Level:      1,
+					Time:       time.Date(2022, 12, 31, 11, 59, 1, 0, time.UTC),
+					LoggerName: "logger_1",
+					Message:    "message 1",
+				},
+			},
+		}, {
+			inLog: zapcore.Entry{
+				Level:      2,
+				Time:       time.Date(2022, 12, 31, 11, 59, 2, 0, time.UTC),
+				LoggerName: "logger_2",
+				Message:    "message 2",
+			},
+			expected: []zapcore.Entry{
+				{
+					Level:      1,
+					Time:       time.Date(2022, 12, 31, 11, 59, 1, 0, time.UTC),
+					LoggerName: "logger_1",
+					Message:    "message 1",
+				},
+				{
```

Comment:
```suggestion
				}, {
```

Reply:
Changed the layout in many places.
But task fmt does as he pleases